lib/localenv/clusterenv.go

```diff
@@ -135,6 +135,7 @@ func newClusterEnvironment(args clusterEnvironmentArgs) (*ClusterEnvironment, er
 		Apps:     apps,
 		Users:    users,
 		StateDir: siteDir,
+		Local:    true,
```

Comment:
This is needed (again) as using `fsm.Logger` with an operator from clusterenv would not log to the correct location.